db/seeds.rb

```diff
@@ -1,8 +1,30 @@
 # frozen_string_literal: true
+
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+require 'csv'
+
+ProduceItem.destroy_all
+ItemSeason.destroy_all
+
+id_arr = []
+
+CSV.foreach(Rails.root.join('lib/produce.csv'), headers: true) do |row|
```

Comment:
Up to taste, since it's easy to understand what's going on here, but I'd generally prefer to name the rows. E.g.:
```rb
produce_name, produce_category, country_code, month_index = row
```

Again, up to you opting for it or not :)

Reply:
from this, we will get produce_name = row[0] and so on?

Reply:
I tried it, but got a validation error `ArgumentError: '["category", "fruit"]' is not a valid category`

Reply:
Ah, I thought it produced an array per iteration. Looking at it I discovered that not. It iteratares through an array of objects (i.e. `#<CSV::Row "name":"abacaxi_havaí" "category":"fruit" "country_code":"BR" "month_index":"1">`).

When we try destructuring, it returns a tuple with header and value (like `["category", "fruit"]`).

An interesting consequence, is that this implies that calling `row.to_h` will give you a hash with the proper keys:
```rb
{"name"=>"abacaxi_havaí", "category"=>"fruit", "country_code"=>"BR", "month_index"=>"1"}
```

So we could use this hash or keep as is. I don't have a preference here.

Reply:
this one confused me a bit, is there anything I could read that might help?